zqd/handlers_test.go

```diff
@@ -398,32 +399,46 @@ func TestPostNDJSONLogs(t *testing.T) {
 			ndjsonio.Rule{"_path", "http", "http_log"},
 		},
 	}
-	_, client, done := newCore(t)
-	defer done()
-	const spaceName = "test"
 
-	_, err := client.SpacePost(context.Background(), api.SpacePostRequest{Name: spaceName})
-	require.NoError(t, err)
+	test := func(t *testing.T, input string) {
```

Comment:
nit: Since `t` is in the enclosing scope, you don't need to pass it again here...